nucliadb/tests/ingest/fixtures.py

```diff
@@ -203,18 +203,20 @@ async def audit():
 
 
 @pytest.fixture(scope="function")
-async def stream_audit(natsd: str):
-    from nucliadb_utils.settings import audit_settings
-
-    audit = StreamAuditStorage(
-        [natsd],
-        audit_settings.audit_jetstream_target,  # type: ignore
-        audit_settings.audit_partitions,
-        audit_settings.audit_hash_seed,
+async def usage(natsd):
+    from nucliadb_utils.settings import usage_settings
+    from nucliadb_utils.utilities import Utility, set_utility
+
+    report_util = KbUsageReportUtility(
+        nats_servers=[natsd], nats_subject=usage_settings.usage_jetstream_subject
+    )
+    await report_util.initialize()
+    report_util.nats_connection_manager.js.publish = AsyncMock(
+        side_effect=report_util.nats_connection_manager.js.publish
     )
-    await audit.initialize()
-    yield audit
-    await audit.finalize()
+    set_utility(Utility.USAGE, report_util)
```

Comment:
You must call `clean_utility` just after the yield to remove the utility from the register. It probably works because I think we have some hard cleanup, but can be problematic